play-services-core/src/main/kotlin/org/microg/gms/accountsettings/ui/WebViewHelper.kt

```diff
@@ -109,7 +111,14 @@ class WebViewHelper(private val activity: AppCompatActivity, private val webView
             }
             Log.d(TAG, "Opening $authUrl")
             webView.post {
-                loadWebViewUrl(authUrl)
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
Please directly refer to the respective constant value of the API level - in this case `21` instead `Build.VERSION_CODES.LOLLIPOP`.